dev_tools/docker_test.py

```diff
@@ -22,6 +22,7 @@ def test_docker_stable():
     assert result.returncode == 0
 
 
+@pytest.mark.skip(reason="fails due to boken dependendencies #6475")
```

Comment:
fyi - it is better to submit a separate tiny PR with this change only so that we keep a single topic per PR.